pkg/apis/hive/v1alpha1/validating-webhooks/clusterdeployment_validating_admission_hook_test.go

```diff
@@ -46,6 +47,40 @@ func clusterDeploymentWithManagedDomain(domain string) *hivev1.ClusterDeployment
 	return cd
 }
 
+func validGCPClusterDeployment() *hivev1.ClusterDeployment {
+	cd := validAWSClusterDeployment()
```

Comment:
Don't start from an AWS clusterdeployment. If there are common things in clusterdeployment that we want to share amongst the validXXXClusterDeployment functions, then pull those into a separate function that all of the validXXXClusterDeployment functions call.

Reply:
done